src/index.html

```diff
@@ -3,18 +3,15 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Page title</title>
+    <title>Event Booster</title>
     <link rel="stylesheet" href="./sass/index.scss" />
     <script type="module" src="./index.js"></script>
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
   </head>
   <body>
-    <div class="events">
-      aaa
-    </div>
     <!-- ❗️❗️❗️ Add partials like so for every page -->
-
-
-    <p>Adrianna</p>
+    <include src="./partials/header.html"></include>
+    <div class="events"></div>
     <include src="./partials/example.html"></include>
```

Comment:
wyskoczył konflikt. W pliku index.html
Przy rozwiązywaniu konfliktów wybierz opcję, żeby wszystko zostało (opcja both).
I pewno to include z example możesz zakomentować, żeby nam się nie pojawiało.